content/docs/glossary/_index.md

```diff
@@ -261,14 +275,16 @@ Therefore ideas, implementation plans or questions can be brought to a SIG for a
 
 For more information, please refer to this [blog post](https://www.giantswarm.io/blog/the-giant-swarm-model-explained)
 
-### Single sign-on (SSO)
```

Comment:
Did you mean to delete this?

Reply:
Oh nvm it was moved.

Reply:
Yeah, it wasn't alphabetical so I fixed it.